python/ray/train/trainer.py

```diff
@@ -143,15 +151,23 @@ def __init__(
                     "request a positive number of `GPU` in "
                     "`resources_per_worker.")
 
+        runtime_env = {
+            "env_vars": {
+                var_name: os.environ.get(var_name, "")
+                for var_name in BACKEND_ENV_VARS
```

Comment:
nit: Instead of always passing it in should we only do so if it is populated? Something like:

```suggestion
                var_name: os.environ[var_name]
                for var_name in BACKEND_ENV_VARS
                if var_name in os.environ
```

Reply:
Good point, done!